services/streammanager/kafka/kafkamanager.go

```diff
@@ -581,7 +594,7 @@ func sendBatchedMessage(ctx context.Context, jsonData json.RawMessage, p produce
 	return 200, returnMessage, returnMessage
 }
 
-func sendMessage(ctx context.Context, jsonData json.RawMessage, p producerManager, topic string) (int, string, string) {
+func sendMessage(ctx context.Context, jsonData json.RawMessage, p producerManager, topic string, config configuration) (int, string, string) {
```

Comment:
remove topic argument, just handle logic using config

Reply:
this is addressed